tnoodle-ui/src/main/components/EventPicker.jsx

```diff
@@ -19,6 +19,7 @@ const mapStateToProps = (store) => ({
     scramblingProgressTarget: store.scramblingProgressTarget,
     scramblingProgressCurrent: store.scramblingProgressCurrent,
     fileZipBlob: store.fileZipBlob,
+    generatingScrambles: store.generatingScrambles,
```

Comment:
[Travis](https://travis-ci.org/github/thewca/tnoodle/builds/742319562#L373) is complaining that this is duplicated. Could you please fix it?

Reply:
Hi, I made a new commit and pushed it onto this fork. Did it work? thanks!